mock/README.md

```diff
@@ -0,0 +1,3 @@
+### Run the server
+
+`json-server db.json --routes routes.json`
```

Comment:
Nit, I'd rather start the json-server through npx.

````suggestion
# External API mock server

## Run the server
The server used to mock the API is `json-server`. You can start it using the following command:

```sh
npx json-server db.json --routes routes.json
```
````